src/index.ts

```diff
@@ -12,6 +12,8 @@ if (cmd === 'run') {
     console.error(error);
     process.exit(1);
   });
+} else if (cmd === 'setup') {
+  await init(false, tool);
```

Comment:
What is the usecase for this? Just wondering as it seems to not be documented anywhere. Having both `setup` and `init` is a code smell to me due to the similar definitions of both verbs.